test/e2e/lifecycle/cluster_upgrade.go

```diff
@@ -177,7 +183,35 @@ var _ = SIGDescribe("etcd Upgrade [Feature:EtcdUpgrade]", func() {
 				defer finalizeUpgradeTest(start, etcdTest)
 				framework.ExpectNoError(framework.EtcdUpgrade(framework.TestContext.EtcdUpgradeStorage, framework.TestContext.EtcdUpgradeVersion))
 			}
-			runUpgradeSuite(f, testFrameworks, testSuite, upgCtx, upgrades.EtcdUpgrade, upgradeFunc)
+			runUpgradeSuite(f, upgradeTests, testFrameworks, testSuite, upgCtx, upgrades.EtcdUpgrade, upgradeFunc)
+		})
+	})
+})
+
+var _ = SIGDescribe("stateful Upgrade [Feature:StatefulUpgrade]", func() {
```

Comment:
This is going to put "[sig-cluster-lifecycle]" in the name when I don't think we're going to own it. Can you use the standard describe and add an owning sig here?